core/src/main/java/org/ethereum/beacon/core/types/Time.java

```diff
@@ -0,0 +1,50 @@
+package org.ethereum.beacon.core.types;
+
+import org.ethereum.beacon.ssz.annotation.SSZSerializable;
+import tech.pegasys.artemis.util.uint.UInt64;
+
+@SSZSerializable(serializeAs = UInt64.class)
+public class Time extends UInt64 implements SafeComparable<Time> {
```

Comment:
I think that `Timestamp` would more precise.

Reply:
`Timestamp` was my initial effort. But this type refers both `timestamp` and `time period` (like `SlotDuration`), so I just named it `Time`

Reply:
Let's leave `Time` for now. I like `Seconds` less :/

Reply:
Making separate `Timestamp` and `Period` types sounds like an overkill to me for existing 3 class usages